packages/web/src/components/molecules/AssetItemOnList/index.tsx

```diff
@@ -146,7 +146,7 @@ export const AssetItemOnList = ({
         )}
         {enableWithdrawHoldersReward ? (
           <Button type="link" onClick={onClick(onClickWithdrawHoldersReward)}>
-            Withdraw
+            holders
```

Comment:
Why did you make this a `holders` ?
This button is there for Withdraw, so I don't think it needed to be changed.

Reply:
Sorry. I miss understand.
I will modifiy again